### ISSUE TITLE
Streaming Audio Maxes Out CPU on FFMPEG

### ISSUE THREAD

**Issue by chriskewl** [retrocomputacion/retrobbs#25]
Using a Raspberrry Pi 4 I see 100% usage on the Python code while streaming the song. The CPU is being hammered when streaming audio is playing but it's the Python doing it and not the ffmpeg.

[Screenshot of htop](https://cdn.discordapp.com/attachments/1065715215414599680/1073880594330693703/qXsvDavkE08yhpGkyGp1PCcd5C2e8TUdkfVZgBtilw_j8cWtr5-lmb64ZVickV8vkTa7Gjol-hZ_3GKiIlcEstfl_AqeAPGj-AGHB6Dhmw-ptPtplY3LnrQi5X41JWQdJvrUjQLgm1lpbB1AiRPqvDpl1wjKJuG5BI_UMihpBNbdfHuI87JSyaI6vCLrdqnXBEsjXJmqgH5_oYnNYKw_yTLZBxjOcka8n8_220HKP37RzbfEy8HHr8uyOa4L4aoScOcikQjOFnBeHVbADmTqYEA6vJfbko6R05Y6dpRlYiPud9UiCWV9mb0RFjSqvm_1yKJjc-qKCqan1sKKVvine6eVZ6IP8TR69UNUIOj2sjrC_Wi52UBC1PD4fV5PswSdyDGQEuGvCQM7REsADkqacU5IpdLszPzrxGaB1rAxSrNlKe1rqw1eXNP1PKxX79uA_zYUoHvWYTtR7Wd3hYTMrv1ENRFRqRIx81Lh4c8v_Xy_F0CxYTUr8Xxf5EOw..png)

**Comment by retrocomputacion** [comment]
Should be fixed in [fc2c1cb](https://github.com/retrocomputacion/retrobbs/commit/fc2c1cb3443afcdcea0d90c30791141262b51f1d)

**Comment by chriskewl** [comment]
Yes but moving the speed from 0.9 to 0.75 helped solve the skipping issue.

**Comment by retrocomputacion** [comment]
Did you also have to reduce the value to 0.75 on the latest version?
